app/views/posts/index.html.erb

```diff
@@ -1 +1,33 @@
-<h1>A list of posts for a given user</h1>
\ No newline at end of file
+<%= render @user %>
+
+<div class = "bio">
+  <h3>Bio</h3>
+  <%= @user.Bio %> 
+</div>
+
+<h1>A list of posts for a given user</h1>
+
+<div class="last-posts">
+  <% @posts.each do |post| %>
+    <%= render post %>
+    <div class="comments">
+      <% post.recent_5_comments.each do |comment| %>
```

Comment:
Uses `post.recent_comments` to render 5 most recent comments in the `app/views/posts/index.html.erb` page 🙌 Great!